dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/kdoc/ResolveKDocLink.kt

```diff
@@ -37,7 +37,7 @@ internal inline fun DRI?.ifUnresolved(action: () -> Unit): DRI? = this ?: run {
  */
 internal fun KtAnalysisSession.resolveKDocTextLinkDRI(link: String, context: PsiElement? = null): DRI? {
```

Comment:
`resolveKDocTextLinkToDRI`

Reply:
I didn't rename this one because it was consistent with the function `resolveKDocTextLinkSymbol` below, but I guess I'll rename it to `resolveKDocTextLinkToSymbol` as well